app/src/test/java/ca/dal/cs/athletemonitor/athletemonitor/TeamTest.java

```diff
@@ -10,33 +10,42 @@
 /**
  * Unit tests for the Team class
  */
-
 public class TeamTest {
```

Comment:
Add @RunWith(AndroidJUnit4.class), clearly android studio notices the tests anyway, but just to be safe

Reply:
I think this is not possible for unit test

Reply:
My bad, thought this was a espresso test, in that case I believe it should be `@RunWith(JUnit4.class)`

Reply:
Ack, i added it to all of the unit test classes